### PR TITLE
Beoble 1497

### PR DESCRIPTION
**Pull request by bamnenim-kryptonium**
Search Page

### CODE REVIEW

File: packages/react/src/helpers/chains.ts [hunk 1]
```diff
@@ -22,6 +23,8 @@ export const getChainImagePath = (chain: ChainType | undefined): string => {
       return typeof BINANCE === 'string' ? BINANCE : (BINANCE as any).src;
     case 'AVALANCHE':
       return typeof AVALANCHE === 'string' ? AVALANCHE : (AVALANCHE as any).src;
+    case 'GNOSIS_CHAIN':
```
Comment (vanilla-james) [inline packages/react/src/helpers/chains.ts:26]: maybe keep `GNOSIS_CHAIN` and `GNOSIS` as the same value for simplicity?

File: packages/react/src/components/Cells/Svg/Icons/index.tsx [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export * from './Send';
```
[the remark this answers is not in the document]
Comment (bamnenim-kryptonium) [reply packages/react/src/components/Cells/Svg/Icons/index.tsx:1]: followed pattern in here
[pancakeswap](https://github.com/pancakeswap/pancake-frontend/tree/develop/packages/uikit/src/components/Svg/Icons)

File: packages/react/src/components/Chat/ChatInput.tsx [hunk 3]
```diff
@@ -22,9 +22,9 @@ import IconButton from '../IconButton';
 import { useBeoble, useGetUserNfts } from '../../hooks';
 import { MdClose } from 'react-icons/md';
 import Spinner from '../Spinner';
-import { ReactComponent as VideoIcon } from '../../assets/svg/file-icon.svg';
-import { ReactComponent as GIFIcon } from '../../assets/svg/giphy.svg';
-import { ReactComponent as AttachmentIcon } from '../../assets/svg/attachments.svg';
+import { ReactComponent as VideoIcon } from '../../assets/svg(deprecated)/file-icon.svg';
```
[the remark this answers is not in the document]
Comment (bamnenim-kryptonium) [reply packages/react/src/components/Chat/ChatInput.tsx:25]: was too lazy to check all svgs and match them with the names in [design system](https://www.figma.com/file/1CPqj0Dqv2AtH7Ffjayj8h/beoble-web-app?type=design&node-id=5702-80883&t=oDWpAgZBzwbsXPYK-0),
so just marked previous svg assets to deprecated and created new svg folder

File: packages/react/src/helpers/chains.ts (hunk 1 above)
Comment (bamnenim-kryptonium) [reply packages/react/src/helpers/chains.ts:26]: But, yes i will match the names!

**Pull request merged**